### PR TITLE
Make is_user_defined nullable so we can store false on oracle

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Comment by nickvergessen** [comment]
More tables to be added

**Comment by nickvergessen** [comment]
Done, ready for review

**Review by blizzz — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

File: apps/user_status/composer/composer/autoload_static.php [hunk 1]
```diff
@@ -43,6 +43,7 @@ class ComposerStaticInitUserStatus
         'OCA\\UserStatus\\Listener\\UserLiveStatusListener' => __DIR__ . '/..' . '/../lib/Listener/UserLiveStatusListener.php',
         'OCA\\UserStatus\\Migration\\Version0001Date20200602134824' => __DIR__ . '/..' . '/../lib/Migration/Version0001Date20200602134824.php',
         'OCA\\UserStatus\\Migration\\Version0002Date20200902144824' => __DIR__ . '/..' . '/../lib/Migration/Version0002Date20200902144824.php',
+        'OCA\\UserStatus\\Migration\\Version1000Date20201111130204' => __DIR__ . '/..' . '/../lib/Migration/Version1000Date20201111130204.php',
```
Comment (MorrisJobke) [inline apps/user_status/composer/composer/autoload_static.php:46]: The big version jump in the migration is okay?

**Comment by MorrisJobke** [comment]
> The autoloaders are not up to date
> Please run: bash build/autoloaderchecker.sh
> And commit the result

Let me commit that.

**Comment by juliushaertl** [comment]
Failure unrelated (mysql timing out after an hour)

File: apps/user_status/composer/composer/autoload_static.php (hunk 1 above)
Comment (nickvergessen) [reply apps/user_status/composer/composer/autoload_static.php:46]: well the app version is 1.0.x so its 1000 for migrations

**Pull request merged**